examples/Makefile

```diff
@@ -0,0 +1,95 @@
+# ***************************************************************************************
+# ***************************************************************************************
+#
+#		Name : 		Makefile
+#		Author :	Paul Robson (paul@robsons.org.uk)
+#		Date : 		12th February 2024
+#		Reviewed :	No
+#		Purpose :	Basic Main Makefile
+#
+# ***************************************************************************************
+# ***************************************************************************************
+
+ifeq ($(OS),Windows_NT)
+include ..\build_env\common.make
+else
+include ../build_env/common.make
+endif
+
+
+all: basicdemos
+	$(CDEL) samples.zip
+	zip -r samples.zip basic-examples C assembly
```

Comment:
the directory name under examples/ is 'basic', not 'basic-examples', though IMHO it should be 'BASIC'